src/bridges/set/IssuanceBridge.sol

```diff
@@ -3,40 +3,62 @@
 pragma solidity >=0.6.10 <=0.8.10;
```

Comment:
Custom errors not available from `0.6.10` should be min `0.8.4`. When doing this, we could also get away with the experimental below.

Reply:
Thanks for pointing that out 👍